AyonCiCd/Project.py

```diff
@@ -418,7 +418,7 @@ def __get_venv_activate_cmd(self, venv_path):
             activate_cmd = f"{activate_script}"
         else:
             activate_script = os.path.join(venv_path, "bin", "activate")
-            activate_cmd = f"source {activate_script}"
+            activate_cmd = f". {activate_script}"
```

Comment:
`source` did not work for me there, got me a `No such file or directory`.
I googled it quickly and found out that some people recommend `.` instead which worked for me. 

Seems to be caused by a SHELL mismatch but honestly I did not investigate it through.

Reply:
okay i found a nice stack overflow post about it and what your experiencing is in line so lets change it. 

https://stackoverflow.com/questions/11027782/virtualenv-venv-bin-activate-vs-source-venv-bin-activate
https://www.thegeekstuff.com/2010/07/execute-shell-script/